woocommerce/woocommerce
# [e2e-utils-playwright] Configure changelogger

### Submission Review Guidelines:

Closes https://github.com/woocommerce/woocommerce/issues/52941

Configure changelogger for the newly added `e2e-utils-playwright` package.

### How to test the changes in this Pull Request:

- See [CI / Validate changelog](https://github.com/woocommerce/woocommerce/actions/runs/11742849821/job/32714444928?pr=52668) before the changelog file was added - it failed.
- See [CI / Validate changelog](https://github.com/woocommerce/woocommerce/actions/runs/11742991911/job/32714895763?pr=52668#step:5:26) after the changelog file was added - it passed.


cc @Sidsector9
